easybuild/easyconfigs/b/Blender/Blender-2.66a-foss-2016.04-trimmed.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'CMakeMake'
+
+name = 'Blender'
+version = '2.66a'
+versionsuffix = '-trimmed'
+
+homepage = 'https://www.blender.org/'
+description = """Blender is the free and open source 3D creation suite. It supports
+ the entirety of the 3D pipeline-modeling, rigging, animation, simulation, rendering,
+ compositing and motion tracking, even video editing and game creation."""
+
+toolchain = {'name': 'foss', 'version': '2016.04'}
+
+sources = ['blender-2.66a.tar.gz']
```

Comment:
Okay, I'll try to make sure that I'm more conscious of not hard coding things especially when I don't have an example to look up upon.